osu.Game/Screens/Edit/Components/PlaybackContainer.cs

```diff
@@ -0,0 +1,168 @@
+﻿// Copyright (c) 2007-2017 ppy Pty Ltd <contact@ppy.sh>.
+// Licensed under the MIT Licence - https://raw.githubusercontent.com/ppy/osu/master/LICENCE
+
+using OpenTK;
+using osu.Framework.Allocation;
+using osu.Framework.Graphics;
+using osu.Framework.Graphics.Containers;
+using osu.Framework.Graphics.UserInterface;
+using osu.Framework.Input;
+using osu.Game.Graphics;
+using osu.Game.Graphics.Sprites;
+using osu.Game.Graphics.UserInterface;
+
+namespace osu.Game.Screens.Edit.Components
+{
+    public class PlaybackContainer : BottomBarContainer
+    {
+        private readonly IconButton playButton;
+
+        private bool lastTrackState;
+
+        public PlaybackContainer()
+        {
+            PlaybackTabControl tabs;
+
+            Children = new Drawable[]
+            {
+                playButton = new IconButton
+                {
+                    Anchor = Anchor.CentreLeft,
+                    Origin = Anchor.Centre,
+                    Scale = new Vector2(1.4f),
+                    IconScale = new Vector2(1.4f),
+                    Icon = FontAwesome.fa_play_circle_o,
+                    Action = play,
+                    Padding = new MarginPadding { Left = 20 }
+                },
+                new OsuSpriteText
+                {
+                    Origin = Anchor.BottomLeft,
+                    Text = "Playback Speed",
+                    RelativePositionAxes = Axes.Y,
+                    Y = 0.5f,
+                    Padding = new MarginPadding { Left = 45 }
+                },
+                new Container
+                {
+                    Anchor = Anchor.BottomLeft,
+                    Origin = Anchor.BottomLeft,
+                    RelativeSizeAxes = Axes.Both,
+                    Height = 0.5f,
+                    Padding = new MarginPadding { Left = 45 },
+                    Child = tabs = new PlaybackTabControl(),
+                }
+            };
+
+            tabs.AddItem(0.25);
+            tabs.AddItem(0.75);
+            tabs.AddItem(1);
+
+            tabs.Current.ValueChanged += newValue => Track.Tempo.Value = newValue;
+        }
+
+        private void play()
+        {
+            if (Track.IsRunning)
+                Track.Stop();
+            else
+                Track.Start();
+        }
+
+        protected override void Update()
+        {
+            base.Update();
+
+            var currentTrackState = Track.IsRunning;
+            if (currentTrackState == lastTrackState)
+                return;
+
+            playButton.Icon = currentTrackState ? FontAwesome.fa_pause_circle_o : FontAwesome.fa_play_circle_o;
+
+            lastTrackState = currentTrackState;

```

Comment:
Not worth it to store the last state here. Icon internally optimises the case where it hasn't changed.